src/test/java/core/basesyntax/strategy/impl/ReturnOperationHandlerTest.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.Test;
+
+class ReturnOperationHandlerTest {
+    private final ReturnOperationHandler returnOperationHandler
+            = new ReturnOperationHandler();
```

Comment:
initialize your services in methods that has annotation BeforeAll (or Before depends on junit version)
it should be called once
fix all test classes accordingly

Reply:
I added "@BeforeAll" and "@BeforeEach" methods to all test classes where possible. Done.